doc/nrf/libraries/bluetooth/mesh/scheduler_srv.rst

```diff
@@ -146,6 +146,14 @@ Extended models
 The Scheduler Server is implemented as a root model.
 When a Scheduler Server model is present on an element, the Scene Server model (see the :ref:`bt_mesh_scene_srv_readme` documentation) shall also be present on the same element.
```

Comment:
```suggestion
When a Scheduler Server model is present on an element, the Scene Server model (see the :ref:`bt_mesh_scene_srv_readme` documentation) or the Generic OnOff Server model (see the :ref:`bt_mesh_onoff_srv_readme` documentation) or both(depends on managed Scheduler Server actions) shall also be present on the same or lower indexed element as the Scheduler Server model.
```

Reply:
ONOFF server is searched on all elements: